File: c/PRAK204-2510817210027-Ahmad_Tibrizi.c
#include <stdio.h>

int main() {
    float jari, tinggi, volume, luas, keliling;
    const float pi = 22.0 / 7.0;

    printf("Masukkan jari-jari: ");
    scanf("%f", &jari);
    printf("Masukkan tinggi: ");
    scanf("%f", &tinggi);

    volume = pi * jari * jari * tinggi;
    luas = 2 * pi * jari * (jari + tinggi);
    keliling = 2 * pi * jari;

    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);

    return 0;
}

File: python/PRAK204-2510817210027-Ahmad_Tibrizi.py
jari = float(input("Masukkan jari-jari: "))
tinggi = float(input("Masukkan tinggi: "))
pi = 22/7
volume = pi * jari * jari * tinggi
luas = 2 * pi * jari * (jari + tinggi)
keliling = 2 * pi * jari

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")